lib/engine_cart/gemfile_stanza.rb

```diff
@@ -11,7 +11,7 @@ def self.gemfile_stanza_text
     # engine_cart: #{EngineCart::VERSION}
     # #{EngineCart.gemfile_stanza_check_line}
     # the below comes from engine_cart, a gem used to test this Rails engine gem in the context of a Rails app.
-    file = File.expand_path("Gemfile", ENV['ENGINE_CART_DESTINATION'] || ENV['RAILS_ROOT'] || File.expand_path("#{EngineCart.destination}", File.dirname(__FILE__)))
+    file = File.expand_path('Gemfile', ENV['ENGINE_CART_DESTINATION'] || ENV['RAILS_ROOT'] || File.expand_path('#{EngineCart.destination}', File.dirname(__FILE__)))
```

Comment:
This whole thing is wrapped in a HEREDOC, and is interpolated when the rake task is run. It isn't interpolated within the Gemfile at all.